code/modules/awaymissions/maploader/reader.dm

```diff
@@ -358,28 +358,30 @@ GLOBAL_DATUM_INIT(_preloader, /datum/dmm_suite/preloader, new())
 /datum/dmm_suite/proc/readlist(text, delimiter = ",")
 	var/list/to_return = list()
 
-	var/position
+	var/delimiter_position
```

Comment:
Could you note where the actual change is instead of a fuck ton of var renames?

Reply:
Just did yes